Source/Renderer/Context.js

```diff
@@ -278,13 +293,39 @@ define([
         this._textureFilterAnisotropic = textureFilterAnisotropic;
         ContextLimits._maximumTextureFilterAnisotropy = defined(textureFilterAnisotropic) ? gl.getParameter(textureFilterAnisotropic.MAX_TEXTURE_MAX_ANISOTROPY_EXT) : 1.0;
 
-        this._vertexArrayObject = getExtension(gl, ['OES_vertex_array_object']);
+        var vertexArrayObject = getExtension(gl, ['OES_vertex_array_object']);
+        this._vertexArrayObject = vertexArrayObject;
+        this.glCreateVertexArray = function() {
```

Comment:
Also `this.glCreateVertexArray` should be `undefined` if both WebGL 2 and the extension are not enabled.